.github/actions/check-submodules/index.js

```diff
@@ -143,8 +143,18 @@ const checkSubmodules = async function () {
             if (nonCrtRepo.test(submodule.name)) {
                 const isOnMain = await isAncestor(diff.thisCommit, 'origin/main', submodule.path);
                 if (!isOnMain) {
-                    core.setFailed(`Submodule ${submodule.name} is using a branch`);
-                    return;
+                    const awslc = /^(aws-lc)$/
+                    if (awslc.test(submodule.name)) {
```

Comment:
I don't know javascript enough but `awslc.test()` seems a bit weird to me. Can we do something like `if (/^(aws-lc)$/.test(submodule.name)) {`?